app/assets/scripts/views/project.js

```diff
@@ -143,7 +149,7 @@ var Project = React.createClass({
                 </ul>
               </div>
               <dl className={'inpage-meta project--' + ontime}>
-                <dt className='inpage-meta__label visually-hidden'>Status</dt>
+                <dt className='inpage-meta__label'>Status <span className='inpage-meta__label--light'>{data.status[lang]}</span></dt>
```

Comment:
This should be in the translation yml files rather than hard-coded, and the english version should have a colon after "Status".